R/estimate_secondary.R

```diff
@@ -40,6 +40,11 @@
 #' use for estimation but not to fit to at the beginning of the time series.
 #' This must be less than the number of observations.
 #'
+#' @param weigh_prior_delays Logical. If TRUE, all delay distribution
```

Comment:
Rather than duplicating we likely want to inherit from one of the functions.

Reply:
We can only do that if we don't want to mention the default in the man page. I felt this trumped the non-repetition. That said, one of `estimate_truncation` and `estimate_secondary` could still inherit from each other.